src/components/DataplanePolicies/DataplanePolicies.vue

```diff
@@ -175,6 +172,30 @@ export default {
           dppName: this.dppName,
         })
 
+        for (const item of items) {
+          item.policyTypes = {}
+
+          // eslint-disable-next-line guard-for-in
```

Comment:
There isn’t really a good reason to prevent unguarded iteration over a value that you know to be a plain object without a complex prototype chain. In my experience, I’ve never seen a stray prototype value leak into iteration and wreak havoc before. I don’t think this rule should apply in general, but tuning linting rules is a story left for another day.